portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +35,38 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         response.getWriter().println(json);
     }
 
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        final String ANONYMOUS = "Anonymous";
```

Comment:
You may want to add a 'static' before this variable or change the naming. https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names